backend/src/models.py

```diff
@@ -1,10 +1,26 @@
-from mongoengine import Document, SortedListField, ReferenceField, StringField
+from mongoengine import (
+    Document,
+    SortedListField,
+    ReferenceField,
+    StringField,
+    IntField,
+    FloatField,
+)
 
 
 class User(Document):
     username = StringField(required=True, max_length=20, unique=True)
     hashed_password = StringField(required=True)
 
 
+class UserStats(Document):
+    username = StringField(required=True, max_length=20, unique=True)
```

Comment:
Consinder using `ReferenceField`, then You would't have to change user creation process.